src/unstract/sdk/adapters/llm/palm/src/static/json_schema.json

```diff
@@ -21,15 +21,15 @@
     "model_name": {
       "type": "string",
       "title": "Model Name",
-      "default": "",
-      "description": "The name of the model to use for this adapter instance. Refer to https://ai.google.dev/models/palm Leave it empty to use the default model."
+      "default": "models/text-bison-001",
+      "description": "The name of the model to use for this adapter instance. Refer https://ai.google.dev/models/palm. Leave it empty to use the default model."
     },
     "num_output": {
       "type": "number",
       "minimum": 0,
       "multipleOf": 1,
       "title": "Max output tokens",
-      "description": "The number of tokens to generate. The higher the number, the longer the response will be. Leave it empty to use the default value."
+      "description": "The number of tokens to generate. This is limited by the maximum supported by the model and will vary from model to model. The higher the number, the longer the response will be. Leave it empty to use the model's maximum."
```

Comment:
@chandrasekharan-zipstack How do we guarantee that we set the max to model's maximum when the field is left empty? Do we have some logic to derive the model's max and setting that while passing it on to llama-index?

Reply:
@gaya3-zipstack this is set by llama index if we pass the `max_tokens` as `None`. That's why we can't assign a default value here

Reply:
Ok. Got it. Thank you!